test/verify/check-system-services

```diff
@@ -40,6 +40,7 @@ class TestServices(MachineCase):
         # Also `reset-failed` as failed units can be still listed with `systemctl` even when all files
         # were removed and daemon reloaded.
         self.restore_dir("/etc/systemd/system", "systemctl daemon-reload && systemctl reset-failed", True)
+        self.restore_dir("/etc/systemd/user", "su - admin -c 'export XDG_RUNTIME_DIR=/run/user/$(id -u admin); systemctl --user daemon-reload && systemctl reset-failed'", True)
```

Comment:
The `reset-failed` is missing `--user`, that's what's breaking the tests.

Reply:
oh awesome you saw this, I was really wondering.